app/models/external_link.rb

```diff
@@ -7,4 +7,8 @@ class ExternalLink < ApplicationRecord
 
   validates :title, presence: true
   validates :link, presence: true, url: true
+
+  def label
```

Comment:
The preferred method for adding a label is via the `label_method`, as defined in the parent model https://github.com/tulibraries/manifold/blob/master/app/models/application_record.rb#L12
```suggestion
  def label_method
```

Reply:
@bibliotechy It appears this way in all the other models, should I change them as well?

Reply:
Hmm, you are right. No, you can disregard this comment